apps/gallery/js/ImageEditor.js

```diff
@@ -223,6 +223,13 @@ var exposureSlider = (function() {
   thumb.addEventListener('touchend', function(e) {
     thumb.classList.remove('active');
   });
+  slider.addEventListener('keypress', function(e) {
+    if (e.keyCode == KeyEvent.DOM_VK_DOWN) {
+      setExposure(currentExposure - 0.25);
+    } else {
+      setExposure(currentExposure + 0.25);
+    }
+  });
```

Comment:
Yes, the key up/down is an unfortunate desktop convention. When you navigate to the slider swipe up and down with one finger to change the value.